docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -2,54 +2,190 @@
 
-These configurations are accessible through `strapi.config.providerURL` and `strapi.config.mainColor`.
+```bash
+HOST=10.0.0.1 NODE_ENV=production yarn start
+# uses host 10.0.0.1
+```
+
+## Server
+
+**Path —** `./config/server.js`.
+
+```js
+module.exports = ({ env }) => ({
+  host: env('HOST', '0.0.0.0'),
+  port: env.int('PORT', 1337),
+});
+```
+
+**Available options**
+
+| Property                                     | Description                                                                                                                                                 | Type          | Default     |
+| -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ----------- |
+| `host`                                       | Host name                                                                                                                                                   | string        | `localhost` |
+| `port`                                       | Port on which the server should be running.                                                                                                                 | integer       | `1337`      |
+| `emitErrors`                                 | Enable errors to be emitted to `koa` when they happen in order to attach custom logic or use error reporting services.                                      | boolean       |             |
+| `proxy`                                      | Proxy configuration                                                                                                                                         | Object        |             |
+| `proxy.enabled`                              | Enable proxy support such as Apache or Nginx.                                                                                                               | boolean       | `false`     |
+| `proxy.ssl`                                  | Enable proxy SSL support.                                                                                                                                   | boolean       |             |
+| `proxy.host`                                 | Host name your proxy service uses for Strapi.                                                                                                               | string        |             |
+| `proxy.port`                                 | Port that your proxy service accepts connections on.                                                                                                        | integer       |             |
+| [`cron`](https://en.wikipedia.org/wiki/Cron) | Cron configuration                                                                                                                                          | Object        |             |
```

Comment:
Shouldn't we link the node module we use instead?

Reply:
I just kept the previous doc but I can make the switch if you want :)